ocp4/profiles/cis.profile

```diff
@@ -95,7 +95,7 @@ selections:
   # 1.3.4 Ensure that the --service-account-private-key-file argument is set as appropriate
     - controller_service_account_private_key
   # 1.3.5 Ensure that the --root-ca-file argument is set as appropriate
-    - controller_service_account_ca
+    - ocp_controller_verify_server_certificate
```

Comment:
This naming convention is a little bit different than the rest in this profile. Does it matter?